python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
Yes but then we end up having to add checks and measures against:
```
lf = LocalFile(namePattern='someFile', localDir='/some/path')
lf.localDir='/some/new/path'
lf.get() # probably won't do what we want if it's not been nulled away
```
Rather than being able to ask the interface to put a file somewhere which works for _all_ files.